Translations update from Weblate

xwikiorgci opened the pull request:
Translations update from [Weblate](https://l10n.xwiki.org/projects/xwiki-contrib/application-discussions-discussions-ui-page-properties/) for XWiki Contrib/Application Discussions - Discussions Macro - Page Properties.



Current translation status:

![Weblate translation status](https://l10n.xwiki.org/widgets/xwiki-contrib/-/application-discussions-discussions-ui-page-properties/horizontal-auto.svg)

If a new locale is introduced by this pull request make sure to follow https://dev.xwiki.org/xwiki/bin/view/Community/L10N/SupportedLocales/#HNewlocalechecklist before applying it.

Commit:
Added translation using Weblate (Spanish)

Commit:
Added translation using Weblate (Spanish)

surli reviewed (changes requested):
Cannot be merged yet, we need the actual translations.

fitojb committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (30 of 30 strings)

Translation: XWiki Contrib/Application Discussions - Discussions Macro - Page Properties
Translate-URL: https://l10n.xwiki.org/projects/xwiki-contrib/application-discussions-discussions-ui-page-properties/es/

fitojb committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (18 of 18 strings)

Translation: XWiki Contrib/Application Discussion - Message Stream Properties
Translate-URL: https://l10n.xwiki.org/projects/xwiki-contrib/application-discussion-message-stream-properties/es/
